jerry-core/ecma/operations/ecma-conversion.cpp

```diff
@@ -489,9 +489,6 @@ ecma_op_from_property_descriptor (const ecma_property_descriptor_t* src_prop_des
   else
   {
     // 4.
-    JERRY_ASSERT (src_prop_desc_p->is_get_defined
-                  || src_prop_desc_p->is_set_defined);
```

Comment:
Is it a valid solution to set the `get_p`/`set_p` to `NULL` and the `is_get_defined`/`is_set_defined` to `true`? IMHO I don't like this kind of hacks.